pages/Documents/ConversationalAI/Third-PartyBots/LivePersonFunctions/advanced-features.md

```diff
@@ -412,3 +412,60 @@ function lambda(input, callback) {
 We can see the above LivePerson function in action below:
 
 <img class="fancyimage" style="width:300px" src="img/faas/faas_richcontent_demo.gif">
+
+### Receiving Last consumer message (Messaging Only)
+
+Third-Party bot now provides a way to add the last consumer message as a part of [the welcome event](third-party-bots-liveperson-functions-basic-content.html#the-welcome-event). When an ongoing conversation gets transferred to a new Agent or Skill, This enhancement will allow brands to respond to the last consumer message uttered as per their needs.
```

Comment:
```suggestion
When an ongoing conversation gets transferred to a bot connected via the Third-Party Bot connector, the connector forwards the last consumer message to the AI vendor as part of the [the welcome event](third-party-bots-amazon-lex-basic-content.html#the-welcome-event).
This allows the bot to react to the last consumer message instead of instantiating a new conversation.
```